swagger.yml

```diff
@@ -224,15 +224,15 @@ paths:
         default:
           $ref: '#/responses/error'
       description: >-
-        [Beta] Returns all environment variables for an account or site. An
+        Returns all environment variables for an account or site. An
         account corresponds to a team in the Netlify UI. To use this endpoint,
         opt in to the beta environment variable experience using the Netlify UI.
```

Comment:
I think we should also update the next line that says `...opt in to the beta environment variable experience...` 

I have conflicting thoughts:

a) We could just remove the second line entirely as this new experience is the new default. Similar to how the docs are written as "new experience-first" (similar to how we say "mobile-first"). This is helpful for new teams as they won't need to stop and wonder if they have to migrate, they can just use the API right away.

b) But, we may still find a lot of current teams checking these reference docs and they may have existing sites that need to be migrated first. So, we could replace it with something like: `If you encounter issues with this endpoint, your site might be using the classic environment variables experience. Migrate your site using the Netlify UI`.  

The second option seems safest and it's what we use in the Docs - it's just long... 

I should know this but can't remember - does the API return anything helpful if you try to use these endpoints on a site that hasn't been migrated?

Reply:
I'm trying to think of a shorter variation that only existing users need to care about: 

`To use this endpoint, your site must no longer be using the classic environment experience. Migrate now with the Netlify UI` ?? 🤔

Reply:
Second one sounds reasonable to me - i think there is only so much we can do with so little text to describe this feature rollout